components/user/UserLink.js

```diff
@@ -19,12 +19,14 @@ export default function UserLink({ link, username, displayStatsPublic }) {
   };
 
   return (
-    <button
-      onClick={() => clickLink()}
-      className="rounded-full border-2 border-gray-200 hover:border-gray-500 hover:shadow-xl p-4 my-2 w-full content-start"
-    >
-      {link.name}{" "}
-      {displayStatsPublic && link.clicks && <span>({link.clicks})</span>}
-    </button>
+    <a className="w-full" rel="noreferrer" target="_blank" href={link.url}>
```

Comment:
I think wrapping the `button` in a `a` is not required, we will do the redirect in the `clickLink` function - there is a `TODO` in there at the moment